sphinx/source/recovery.rst

```diff
@@ -1,105 +1,122 @@
 
-        Members->>+Primary: setRecoveryNodes RPC (new network configuration)
+        Note over Operators, Node 2: Operators select Node 2 to start the new network
 
-        Members->>+Backup: join network
-        Backup->>+Primary: join network
-        Backup-->>Members: join network response
+        Operators->>+Node 3: cchost join --network-cert-file=Network Certificate --target-rpc-address=ip2:port2
+        Node 3->>+Node 2: Join network (over TLS)
+        Node 2-->>Node 3: Network Secrets (over TLS)
 
-        Note over Primary, Backup: Part of Public Network
+        Note over Node 3: Part of Public Network
 
 Phase 2: Unsealing secrets and recovering private transactions
 --------------------------------------------------------------
 
-Once the public crash-fault tolerant network is established, members are allowed to vote to confirm that the configuration of the new network is suitable to complete the recovery protocol. The first member proposes to recover the network and passes the sealed network secrets file to the new network:
+Once the public crash-fault tolerant network is established, members are allowed to vote to confirm that the configuration of the new network is suitable to complete the recovery protocol. The first member proposes to recover the network, passing the sealed network secrets file to the new network:
 
 .. code-block:: bash
 
-    $ memberclient accept_recovery --sealed-secrets=/path/to/sealed/secrets/file --cert=/path/to/member1/cert --privk=/path/to/member1/private/key --server-address=primary_rpc_ip:primary_rpc_port --ca=/path/to/new/network/cert
+    $ memberclient --cert=/path/to/member1/certificate --privk=/path/to/member1/private/key
+    --rpc-address=node2_rpc_ip:node2_rpc_port --ca=/path/to/new/network/certificate
+    accept_recovery --sealed-secrets=/path/to/sealed/secrets/file
 
 If successful, this commands returns the proposal id that can be used by other members to submit their votes:
 
 .. code-block:: bash
 
-    $ ./memberclient vote --accept --cert=/path/to/member2/cert --privk=/path/to/member2/private/key --server-address=primary_rpc_ip:primary_rpc_port --id=proposal_id --ca=/path/to/new/network/cert
+    $ memberclient --cert=/path/to/member2/certificate --privk=/path/to/member2/private/key
+    --rpc-address=node2_rpc_ip:node2_rpc_port --ca=/path/to/new/network/certificate
+    vote --accept --proposal-id=proposal_id
 
-Once a quorum of members (defined by the constitution rules but typically, a majority of members) have agreed to recover the network, the network secrets are unsealed and the recovery of the private entries of the ledger is automatically started.
+Once a quorum of members (defined by the :ref:`governance` rules but typically, a majority of members) have agreed to recover the network, the network secrets are unsealed and the recovery of the private entries of the ledger is automatically started.
```

Comment:
`define by the governance rules, but typically a majority of members`

I'm also not sure about "is automatically started". Perhaps "and each node begins recovery of the private entries".

Do we specify elsewhere that _private_ recovery involves decrypting entries, releasing secret information into a network of trusted nodes?

Reply:
The sentence in bracket no longer exists since I have moved the definition of "quorum" to the glossary. 

I have also made the description of the recovery phases (at the top of the page) more accurate.